packages/manager/modules/dbaas-logs/src/logs/order/logs-order.html

```diff
@@ -0,0 +1,127 @@
+<div class="row p-4">
+    <div class="col-lg-9 p-4">
+        <h1 data-translate="logs_order_title" class="m-0"></h1>
```

Comment:
Why not considering using [`<oui-header`](https://ovh.github.io/ovh-ui-kit/?path=/story/design-system-components-header-webcomponent--default)?